<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSystemBrowser</class>
 <widget class="QDockWidget" name="FileSystemBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File System Browser</string>
  </property>
  <widget class="QWidget" name="contents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignLeft">
     <widget class="QWidget" name="buttons" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="parentDirectoryButton">
         <property name="toolTip">
          <string>Up</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/actions/filesystem_parent_directory.png</normaloff>icons/actions/filesystem_parent_directory.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="homeDirectoryButton">
         <property name="toolTip">
          <string>Home</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/actions/filesystem_home_directory.png</normaloff>icons/actions/filesystem_home_directory.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="projectDirectoryButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Project Directory</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/actions/filesystem_project_directory.png</normaloff>icons/actions/filesystem_project_directory.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="activeFileDirectoryButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Locate active file</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/actions/filesystem_active_file_directory.png</normaloff>icons/actions/filesystem_active_file_directory.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="pathBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="view">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>view</tabstop>
  <tabstop>parentDirectoryButton</tabstop>
  <tabstop>homeDirectoryButton</tabstop>
  <tabstop>projectDirectoryButton</tabstop>
  <tabstop>activeFileDirectoryButton</tabstop>
  <tabstop>pathBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
